src/Console/DuskCommand.php

```diff
@@ -117,7 +117,7 @@ protected function withDuskEnvironment($callback)
         return tap($callback(), function () {
             $this->removeConfiguration();
 
-            if (file_exists(base_path($this->duskFile()))) {
+            if (file_exists(base_path($this->duskFile())) && file_exists(base_path('.env.backup')))) {
```

Comment:
Let's hope that will get merged. Afterwards I'll try to get my `phpunit.dusk.xml` thing from #154 merged. I probably should have made two PRs instead of a combined one :laughing: